pyproject.toml

```diff
@@ -144,9 +144,9 @@ readme = { file = ['README.md'], content-type = 'text/markdown' }
 [build-system]
 build-backend = 'setuptools.build_meta'
 requires = [
-    'setuptools ~= 66.0',
-    'setuptools_scm[toml] ~= 7.1.0',
-    'wheel ~= 0.38.4',
```

Comment:
These particular ones are a safety net, we don't want people using tools _older_ than these for `pyproject.toml`, namely `setuptools`. Might bode well to just change these to `>=`.